scripts/ci_cli.sh

```diff
@@ -6,23 +6,26 @@ ERC721_HANDLER="0x21605f71845f372A9ed84253d2D024B7B10999f4"
 ERC721_RESOURCE_ID="0x00000000000000000000002B6Ab4b880A45a07d83Cf4d664Df4Ab85705Bc0700"
 ERC721_CONTRACT="0x2B6Ab4b880A45a07d83Cf4d664Df4Ab85705Bc07"
 
+GAS_LIMIT=6721975
+GAS_PRICE=20000000000
```

Comment:
are these numbers arbitrary?

Reply:
They are just what ganache uses as defaults. I could have done the inverse and changed ganache to use different params, but I think it's better to specify here, should we ever want to run this script against geth which has different default params.